app/seed-data/emissions_factors/scripts/ghgprotocol_transportation.py

```diff
@@ -418,12 +418,12 @@ def save_to_csv(fl, data):
     # Methodology
     # =================================================================
     methodologies = [
-        "fuel_sales",
+        "fuel-sales",
         "geographic",
-        "induced_activity_1",
-        "induced_activity_2",
-        "resident_activity",
-        "movement_driver",
+        "induced-activity-1",
+        "induced-activity-2",
+        "resident-activity",
+        "movement-driver",
         "TBD",
```

Comment:
Do we still need the TBD here?

Reply:
Probably not, I think that's just a place holder let me filter it out so we don't load unnecessary data